lib/_stream_readable.js

```diff
@@ -660,7 +661,7 @@ Readable.prototype.on = function(ev, fn) {
     this.resume();
   }
 
-  if (ev === 'readable' && this.readable) {
+  if (ev === 'readable') {
```

Comment:
> Since the second condition was always evaluating to true

That's not necessarily [the case](https://github.com/iojs/io.js/blob/v1.x/lib/net.js#L146-L150), different subclasses may be [manipulating](https://github.com/iojs/io.js/blob/v1.x/lib/_tls_legacy.js#L441) that attribute. This may end up changing the behavior of `process.stdout.on('readable')`, or adding a readable listener to a destroyed `TLSSocket`.